docs/installation.rst

```diff
@@ -56,197 +59,193 @@ These examples use conda environments.
+                pip install -r devtools/dev-requirements.txt
+                # finally, install the gpu-compatible JAX that matches the version needed by the DESC requirements
+                # It is important to NOT use the --upgrade or -U flag here! otherwise you may get incompatible JAX versions
+                pip install "jax[cuda12]"
 
+            Tested and confirmed to work on the Della and Stellar clusters at Princeton as of January 30, 2025.
 
-RAVEN (IPP, Germany)
-++++++++++++++++++++++++++++++
-These instructions were tested and confirmed to work on the RAVEN cluster at IPP on Aug 18, 2024
 
-Create a conda environment for DESC
+        .. dropdown:: RAVEN (IPP, Germany)
 
-.. code-block:: sh
+            These instructions were tested and confirmed to work on the RAVEN cluster at IPP on Aug 18, 2024
 
-    module load anaconda/3/2023.03
-    CONDA_OVERRIDE_CUDA="12.2" conda create --name desc-env "jax==0.4.23" "jaxlib==0.4.23=cuda12*" -c conda-forge
-    conda activate desc-env
+            Create a conda environment for DESC
 
-Clone DESC
+            .. code-block:: sh
 
-.. code-block:: sh
+                module load anaconda/3/2023.03
+                CONDA_OVERRIDE_CUDA="12.2" conda create --name desc-env "jax==0.4.23" "jaxlib==0.4.23=cuda12*" -c conda-forge
+                conda activate desc-env
 
-    git clone https://github.com/PlasmaControl/DESC
-    cd DESC
+            Clone DESC
 
-In the requirements.txt file, change the scipy version from
+            .. code-block:: sh
 
-.. code-block:: sh
+                git clone https://github.com/PlasmaControl/DESC
+                cd DESC
 
-    scipy >= 1.7.0, < 2.0.0
+            In the requirements.txt file, change the scipy version from
 
-to
+            .. code-block:: sh
 
-.. code-block:: sh
+                scipy >= 1.7.0, < 2.0.0
 
-    scipy >= 1.7.0, <= 1.11.3
+            to
 
-Install DESC
+            .. code-block:: sh
 
-.. code-block:: sh
+                scipy >= 1.7.0, <= 1.11.3
 
-    # installation for users
-    pip install --editable .
-    # optionally install developer requirements (if you want to run tests)
-    pip install -r devtools/dev-requirements.txt
+            Install DESC
 
+            .. code-block:: sh
 
-On Clusters with IBM Power Architecture
-***************************************
+                # installation for users
+                pip install --editable .
+                # optionally install developer requirements (if you want to run tests)
+                pip install -r devtools/dev-requirements.txt
 
-If pre-built JAX binaries are not available, you will first need to build JAX from source.
-More info can be found here: https://jax.readthedocs.io/en/latest/developer.html
 
-These instructions were tested and confirmed to work on the Traverse supercomputer at Princeton as of Nov. 6, 2023.
+        .. dropdown:: On Clusters with IBM Power Architecture
```

Comment:
i think we can probably get rid of this since Traverse is closed, and the version quoted is over a year old anyways